addons/website_sale/static/src/js/website_sale.editor.js

```diff
@@ -448,6 +448,7 @@ options.registry.WebsiteSaleProductsItem = options.Class.extend({
      */
     createRibbon(previewMode, widgetValue, params) {
         this.saveMethod = 'create';
+        this.setRibbon(false);
```

Comment:
Isn't this enough? Tried locally and seems to work
```suggestion
        this._resetRibbonDummy();
```

Reply:
@rdeodoo It doesnt work on my side.. I still have the same issue explained in the commit message

Reply:
Not sure how did I test it the other day, but it indeed does not work :shrug: